src/datatype.rs

```diff
@@ -40,91 +40,102 @@ pub enum DataType {
     Empty,
 }
 
-impl DataType {
-    /// Assess if datatype is empty
-    pub fn is_empty(&self) -> bool {
+/// An enum to represent all different data types that can appear as
+/// a value in a worksheet cell
+impl DataTypeTrait for DataType {
+    fn is_empty(&self) -> bool {
         *self == DataType::Empty
     }
-    /// Assess if datatype is a int
-    pub fn is_int(&self) -> bool {
+    fn is_int(&self) -> bool {
         matches!(*self, DataType::Int(_))
     }
-    /// Assess if datatype is a float
-    pub fn is_float(&self) -> bool {
+    fn is_float(&self) -> bool {
         matches!(*self, DataType::Float(_))
     }
-    /// Assess if datatype is a bool
-    pub fn is_bool(&self) -> bool {
+    fn is_bool(&self) -> bool {
         matches!(*self, DataType::Bool(_))
     }
-    /// Assess if datatype is a string
-    pub fn is_string(&self) -> bool {
+    fn is_string(&self) -> bool {
         matches!(*self, DataType::String(_))
     }
 
-    /// Try getting int value
-    pub fn get_int(&self) -> Option<i64> {
+    #[cfg(feature = "dates")]
+    fn is_duration(&self) -> bool {
+        matches!(*self, DataType::Duration(_))
+    }
```

Comment:
Why is this (and similar methods) under the dates feature? This feature is needed only for converting excel date to chrono

Reply:
Because the `as_{datetime,duration}` are under the `dates` feature as well, so it was mainly for consistency.